sdk/storage/azure-storage-file-share/azure/storage/fileshare/_file_client.py

```diff
@@ -1474,14 +1474,13 @@ def get_ranges(  # type: ignore
         return [{'start': file_range.start, 'end': file_range.end} for file_range in ranges.ranges]
 
     @distributed_trace
-    def get_ranges_diff(  # type: ignore
-            self,
-            previous_sharesnapshot,  # type: Union[str, Dict[str, Any]]
-            offset=None,  # type: Optional[int]
-            length=None,  # type: Optional[int]
-            **kwargs  # type: Any
-            ):
-        # type: (...) -> Tuple[List[Dict[str, int]], List[Dict[str, int]]]
+    def get_ranges_diff(
+        self,
+        previous_sharesnapshot: Union[str, Dict[str, Any]],
+        offset: Optional[int] = None,
+        length: Optional[int] = None,
+        **kwargs: Any
+    ) -> Tuple[List[Dict[str, int]], List[Dict[str, int]]]:
```

Comment:
Can we make `support_rename` a named keyword after a `*,`. Will require a couple of changes likely.